.github/verible.waiver

```diff
@@ -3,4 +3,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 waive --rule=explicit-parameter-storage-type --location="src/memory_island_core.sv" --line=43
-waive --rule=explicit-parameter-storage-type --location="src/axi_memory_island_wrap.sv" --line=54
+waive --rule=explicit-parameter-storage-type --location="src/axi_memory_island_wrap.sv" --line=59
```

Comment:
Are you tracking this manually? Might be easier to inline waive in you source file.

Reply:
Updated